dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala

```diff
@@ -111,9 +116,13 @@ object EnceladusRestDAO extends EnceladusDAO {
   }
 
   def postLogin(username: String, password: String) = {
```

Comment:
I know it is a pain, but could you annotate this method

Reply:
np